Sources/UI/badge/DJLAppBadgeManager.mm

```diff
@@ -227,14 +284,23 @@ - (void) _updateNewEmailsBadge
     }
     if (hasUnseen) {
         [[NSApp dockTile] setBadgeLabel:@" "];
+        if (showStatusItem) {
+            _statusItem.button.imagePosition = NSImageLeft;
+            _statusItem.button.title = @"!";
```

Comment:
Do you think we can use a unicode circle instead?
I'm even wondering if we could turn the icon blue when there's a new mail.
Let's start with it.

The blue color (or any alternative color) on the menubar icon would be a great improvement.

Reply:
I think it comes down to personal preference and you should choose what you like best.

I tried a bunch of characters including initially various bullets (circles). In the end, I liked the way "!" worked best. It sits close to the icon so they look like they go together and because it is skinny it doesn't seem too loud. The bullets seemed to me to be either too light or too heavy or were spaced too far from the icon. I though they would work better than they did.

With a monochrome template image the o/s takes care of drawing it inverted if the user has the dark menu bar. The o/s also draws it properly if the user has multiple screens and the icons need to be drawn dimmed on the inactive menu bars. A colored icon might stick out from all the other icons since macOS has gone with its current flat monochrome style. Would a color work as an indicator for a user with color blindness?